apps/meteor/client/views/room/MessageList/lib/isMessageSequential.spec.ts

```diff
@@ -135,4 +135,16 @@ describe('isMessageSequential', () => {
 		};
 		expect(isMessageSequential(current, previous, TIME_RANGE_IN_SECONDS)).to.be.false;
 	});
+
+	it('should return false if previous message is thread preview', () => {
+		const previous: IThreadMessage = {
+			...baseMessage,
+			tmid: 'threadId',
+			tshow: true,
+		};
+		const current: IMessage = {
+			...baseMessage,
+		};
+		expect(isMessageSequential(current, previous, TIME_RANGE_IN_SECONDS)).to.be.false;
+	});
 });
```

Comment:
Hi @filipemarins , if you check the file `.mocha.client.js` you will notice that just test files that are inside `tests/unit/**/*` are included in the pipeline, so you need to change to the expected path.
<img width="462" alt="Screen Shot 2022-09-20 at 8 18 40 AM" src="https://user-images.githubusercontent.com/20212776/191244518-6334a559-6a88-497d-9cbb-d24d5fef0ba8.png">

I already did in [this PR](https://github.com/RocketChat/Rocket.Chat/pull/26751) but it's not merged yet.